tests/src/JIT/Directed/rvastatics/RVAOrderingTest.ilproj

```diff
@@ -2,17 +2,37 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <AssemblyName>$(MSBuildProjectName)</AssemblyName>
     <SchemaVersion>2.0</SchemaVersion>
     <ProjectGuid>{95DFC527-4DC1-495E-97D7-E94EE1F7140D}</ProjectGuid>
     <OutputType>Exe</OutputType>
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\</SolutionDir>
     <CLRTestPriority>1</CLRTestPriority>
+
+    <!-- Test relies on rva_statics support see #2451 -->
+    <DisableProjectBuild>true</DisableProjectBuild>
```

Comment:
This disables the project from building, and I believe disables the test from running.

Is this desired/applicable considering the test was just added?

Reply:
Also, it looks like some bug fixes have gone in related to https://github.com/dotnet/coreclr/issues/2451; is the issue still applicable to the other disabled tests?

Reply:
You are right. The test passes when i run it locally (unlike the neighboring ones which fail for me). I'll remove this.